server/sonar-server/src/main/java/org/sonar/server/qualitygate/ws/UpdateConditionAction.java

```diff
@@ -72,14 +77,11 @@ public void define(WebService.NewController controller) {
 
   @Override
   public void handle(Request request, Response response) {
-    userSession.checkPermission(QUALITY_GATE_ADMIN);
+    userSession.checkLoggedIn().checkOrganizationPermission(defaultOrganizationProvider.get().getUuid(), QUALITY_GATE_ADMIN);
```

Comment:
same comment as earlier, addition of checkLoggedIn() introduces a change in behavior which is not mandatory